datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/common/perf_counters.yaml

```diff
@@ -0,0 +1,122 @@
+    https://docs.microsoft.com/en-us/windows/win32/perfctrs/about-performance-counters
+
+    The top-level keys are the names of the desired performance objects:
+
+      metrics:
+        System:
+          <OPTION_1>: ...
+          <OPTION_2>: ...
+        LogicalDisk:
+          <OPTION_1>: ...
+          <OPTION_2>: ...
+
+    The available performance object options are:
+
+      name (required): This becomes the prefix of all metrics submitted for each counter.
+      counters (required): This is the list of counters to collect.
+      tag_name: This is the name of the tag used for instances. For example, if the tag name for
+                the `LogicalDisk` performance object is `disk`, a possible tag would be `disk:C:`.
+                If not set, the default tag name is `instance`.
+      include: This is the list of regular expressions used to select which instances to monitor.
+               If not set, all instances are monitored.
+      exclude: This is the list of regular expressions used to select which instances to ignore.
+               If not set, no instances are ignored. Note: `_Total` instances are always ignored.
+      instance_counts: This is a mapping used to select the count of instances to submit, where each
+                       key is a count type and the value is the metric name to use, ignoring `name`.
+                       The `total` count type represents the total number of encountered instances.
+                       The `monitored` count type represents the number of monitored instances after
+                       `include`/`exclude` filtering. The `unique` count type represents the number
+                       of unique instance names that are monitored.
+      use_localized_counters: Whether or not performance object and counter names should refer to their
+                              locale-specific versions rather than by their English name. This overrides
+                              any defined value in `init_config`.
+
+    The key for each counter object represents the name of the desired counter.
+    Counters may be defined in 2 ways:
+
+    1. If a value is a string, then it represents the suffix of the sent metric name. For example:
+
+         counters:
+         - '% Free Space': usable
+         - Current Disk Queue Length: queue_length.current
+
+    2. If a value is a mapping, then it must have a `name` key that represents the suffix of the
+       sent metric name. For example:
+
+         counters:
+         - '% Free Space':
+             name: usable
+         - Current Disk Queue Length:
+             name: queue_length.current
+
+    The available counter options are:
+
+      type: This represents how the metric is handled, defaulting to `gauge`.
+      average: When there are multiple values for the same instance name (e.g. multiple processes
+               spawned with the same name) the check submits the sum. Setting this option to `true`
```

Comment:
Can you document `time_elapsed`?

Reply:
That will be part of the upcoming tutorial which we'll link here, it's just too much content 😅

Reply:
```suggestion
               spawned with the same name, the check submits the sum. Setting this option to `true`
```